### PR TITLE
Geojson integration

### PR DESCRIPTION
**Pull request by loicgasser**
https://github.com/geoadmin/mf-geoadmin3/pull/2228

### CODE REVIEW

File: chsdi/locale/it/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -1086,7 +1086,7 @@ msgid "ch.bafu.bundesinventare-amphibien_anhang4"
 msgstr "Anfibi allegato 4"
 
 msgid "ch.bafu.bundesinventare-amphibien_anhoerung"
-msgstr "Batraciens consultation 2015"
```
Comment (loicgasser) [inline chsdi/locale/it/LC_MESSAGES/chsdi.po]: @AFoletti Any idea why we lost this tranlsation?

**Comment by loicgasser** [comment]
@procrastinatio and @gjn 

As this PR is not likely to change much, could you please review the integration of the geojson layers?

Should I leave the iframe example? I need to adapt to prod url but otherwise it works fine.

**Comment by loicgasser** [comment]
@AFoletti Could we have transparent background for this legend as well please?
https://github.com/geoadmin/mf-chsdi3/blob/gal_geojson/chsdi/static/images/legends/ch.bafu.hydroweb-messstationen_zustand_de.png

**Comment by gjn** [comment]
I'll have a closer look tomorrow.

File: chsdi/static/doc/examples/geoadmin_iframe.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      }
+      .title {
+        margin-top: -4px;
+        font-weight: bold;
+      }
+      </style>
+  </head>
+  <body>
+    <h1>GeoAdmin Iframe example</h1>
+    <iframe src="http://mf-geoadmin3.dev.bgdi.ch/ltgal/embed.html?X=187027.34&Y=650640.77&zoom=2&lang=fr&topic=dev&bgLayer=ch.swisstopo.pixelkarte-grau&catalogNodes=9900&layers=ch.bafu.hydroweb-messstationen_gefahren,ch.bafu.hydroweb-warnkarte_regional" name="IframeGeoadmin"></iframe>
```
Comment (gjn) [inline chsdi/static/doc/examples/geoadmin_iframe.html]: URL to be adapted

**Comment by gjn** [comment]
This is very good. Maybe 2 things missing, as discussed:
- Documentation (for users who want to use those json layers, as they will be published in the FAQ as usable layers)
- Integration to API ( @kandre is taking care of that, I believe)

Also, I worry a bit about failing tests when putting this to prod.

**Comment by AFoletti** [comment]
hm, tinypng does not make the background transparent for this one. Weird

**Comment by ponceta** [comment]
Maybe source of the data?

**Comment by AFoletti** [comment]
yep... was adapted by Pauline and the transparency went lost.
I've regenerated the legends. Coming soon :)

File: chsdi/locale/en/LC_MESSAGES/chsdi.po [hunk 3]
```diff
@@ -1086,7 +1086,7 @@ msgid "ch.bafu.bundesinventare-amphibien_anhang4"
 msgstr "Amphibians appendix 4"
 
 msgid "ch.bafu.bundesinventare-amphibien_anhoerung"
-msgstr "Amphibien - Anhörung 2015"
+msgstr ""
```
Comment (gjn) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: @AFoletti lost translation?

File: chsdi/static/doc/examples/geoadmin_iframe.html (hunk 2 above)
Comment (gjn) [inline chsdi/static/doc/examples/geoadmin_iframe.html]: @loicgasser Could you update the URL here?

**Comment by gjn** [comment]
@AFoletti @kandre @loicgasser Seems like this PR is not yet fully complete. Could you please finish it up?

File: chsdi/static/doc/examples/geoadmin_iframe.html (hunk 2 above)
Comment (loicgasser) [inline chsdi/static/doc/examples/geoadmin_iframe.html]: It won't work until chsdi and geoadmin are deployed. I don't think we can avoid it unless I make a template...

File: chsdi/tests/integration/test_mapservice.py [hunk 4]
```diff
@@ -465,6 +465,18 @@ def test_layersconfig_valid_topic_all(self):
         self.failUnless('label' in resp.json['ch.swisstopo.pixelkarte-farbe'])
         self.failUnless('background' in resp.json['ch.swisstopo.pixelkarte-farbe'])
 
+    def test_layersconfig_geojson_layer(self):
+        resp = self.testapp.get('/rest/services/all/MapServer/layersConfig', status=200)
+        self.failUnless(resp.content_type == 'application/json')
+        jsonData = resp.json
+        if 'ch.bafu.hydroweb-messstationen_gefahren' in jsonData:
+            layer = jsonData['ch.bafu.hydroweb-messstationen_gefahren']
```
Comment (procrastinatio) [inline chsdi/tests/integration/test_mapservice.py]: Live with your time _failUnless_ is deprecated (_assertTrue_)

**Comment by procrastinatio** [comment]
Very nice, but I really do not understand the use case of your example (ok, it does not work without data):
So we have geojson data from _data.geo.admin.ch_ which are integrated into _map.geo.admin.ch_ via _api3.geo.admin.ch_ to be displayed in an _iframe_ while having some interaction in the _parent page_ ? Is there so easier way ?
If you want to keep it, use the standard geoadmin visual identitiy...

Why do we have the geojson data in 'data.geo.admin.ch' but the style elsewhere (mf-chsdi3) ?

**Pull request merged**